src/components/select/simple-select/simple-select.spec.js

```diff
@@ -74,18 +74,18 @@ describe('SimpleSelect', () => {
     });
   });
 
-  describe('when the preventFocusAutoOpen prop is not set', () => {
+  describe('when the openOnFocus prop not set', () => {
```

Comment:
Think the description needs updating
```suggestion
  describe('when the openOnFocus prop is set', () => {
```